tests/robustness/README.md

```diff
@@ -33,6 +33,30 @@ Upon failure tests generate a report that can be used to attribute whether failu
     * `RESULTS_DIR` - to change location where results report will be saved.
     * `PERSIST_RESULTS` - to persist the results report of the test. By default this will not be persisted in the case of a successful run.
 
+## Re-evaluate existing report
+
+Robustness test validation is constantly changing and improving.
+Errors in etcd model could be causing false positives, which makes the ability to re-evaluate the reports after we fix the issue important.
+
+> Note: Robustness test report format is not stable, and it's expected that not all old reports can be re-evaluated using the newest version.
+
+1. Identify location of the robustness report.
+
+   For local runs this would be by identifying log line, in the following example that would be `/tmp/TestRobustnessExploratory_Etcd_HighTraffic_ClusterOfSize1`:
+   ```
+   logger.go:146: 2024-04-08T09:45:27.734+0200 INFO    Saving robustness test report   {"path": "/tmp/TestRobustnessExploratory_Etcd_HighTraffic_ClusterOfSize1"}
+   ```
+
+   For remote runs on CI you need to go to the Github Actions page, download one of the Artifacts, extract it locally and pick one of the directories located within:
+
+   ![github actions artifact](./github_actions_artifacts.png)
+
+> Note: By default robustness test report is only generated for failed test.
+
+2. Copy the robustness report directory into the `testdata` directory.
```

Comment:
```suggestion
2. Copy the robustness report directory into the `testdata` directory. These directories will be prefixed by `TestRobustness` each representing a scenario. A directory will only have reports if that scenario has failed and if you would like to run this scenario locally, copy the content of that directory into `testdata`.
```

Reply:
This is inaccurate, you are meant to copy the directory, not the contents.

Reply:
Edit this based on https://github.com/etcd-io/etcd/pull/17733#discussion_r1555704118

Reply:
Please see the latest changes.